Makefile

```diff
@@ -1,8 +1,15 @@
 # make file to hold the logic of build and test setup
-ZK_VERSION ?= 3.4.14
-
-ZK = zookeeper-$(ZK_VERSION)
-ZK_URL = "https://archive.apache.org/dist/zookeeper/$(ZK)/$(ZK).tar.gz"
+ZK_VERSION ?= 3.5.6
```

Comment:
Perhaps, worth moving this out into a separate script, e.g. "installzk.sh"

Reply:
Yeah i agree on both accounts